Display title as Loading when info is not ready

epicfaace opened the pull request:
Fixes #1661.

epicfaace committed:
change placeholder to Loading...

yipenghe commented:
The placeholder doesn't seem to be working due to #1689 , plus wouldn't this cause untitled worksheets all display 'Loading'?

yipenghe committed:
Merge branch 'frontend' into untitled

yipenghe commented:
I think for this, we should do the following: check 
1. if this.info && this.info.title => set to info.title 
2. if this.info && !this.info.title => leave undefined (actually no title set before)
3. no this.info => Loading (info data isn't ready yet)

yipenghe reviewed (commented):
This should do the trick

yipenghe commented:
Also would be good to have #1693 when info is not ready, maybe first merge that PR and then add {!info && \<Loading />} to Worksheet.js after {this.state.updating && \<Loading />}

yipenghe committed:
Update frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

Would make interface nicer to use

yipenghe committed:
Update frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

removing the placeholder, just an emptied input for untitled worksheets should be good enough

yipenghe committed:
Merge branch 'frontend' into untitled

yipenghe committed:
Merge branch 'untitled' of https://github.com/codalab/codalab-worksheets into untitled
